series/type-string.go

```diff
@@ -104,6 +108,14 @@ func (e stringElement) Bool() (bool, error) {
 	return false, fmt.Errorf("can't convert String \"%v\" to bool", e.e)
 }
 
+func (e stringElement) Time() (time.Time, error) {
+	t, err := time.Parse(timeformat, e.e)
+	if err != nil {
+		return time.Time{}, fmt.Errorf("can't convert String to time.Time")
```

Comment:
This error message might be misleading, since you *can* convert String to Time, it's just that this particular string has an error:

`"can't convert to time.Time: %v", err`